code/modules/recycling/conveyor2.dm

```diff
@@ -64,19 +68,24 @@
 		if(NORTHEAST)
 			forwards = EAST
 			backwards = SOUTH
-		if(NORTHWEST)
-			forwards = SOUTH
-			backwards = WEST
 		if(SOUTHEAST)
 			forwards = NORTH
 			backwards = EAST
 		if(SOUTHWEST)
 			forwards = WEST
 			backwards = NORTH
+		if(NORTHWEST)
+			forwards = SOUTH
+			backwards = WEST

```

Comment:
I just copied this whole block over from TG. They were out of order there, I believe. One of them was incorrect and throwing things off the belt, I've since gone through carefully and made sure they're all correct now - but I'll put these back in the unchanged order.